portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -26,20 +34,39 @@
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
-static ArrayList<String> myarraylist = new ArrayList<String>(); 
+//static ArrayList<String> myarraylist = new ArrayList<String>(); 
```

Comment:
nit: You can safely remove this line. I don't think you'll need this in the future.